application/tests/unit/common/models/MethodTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+namespace tests\unit\common\models;
+
+use common\models\Method;
+use common\models\User;
+use yii\codeception\DbTestCase;
+
+use tests\unit\fixtures\common\models\UserFixture;
+use tests\unit\fixtures\common\models\MethodFixture;
+
+/**
+ * Class MethodTest
+ * @package tests\unit\common\models
+ * @method User users($key)
+ * @method Method methods($key)
+ */
+class MethodTest extends DbTestCase
+{
+    public function fixtures()
+    {
+        return [
+            'users' => UserFixture::className(),
+            'methods' => MethodFixture::className(),
+        ];
+    }
+
+    public function testDefaultValues()
+    {
+        $user1 = $this->users('user1');
+        $method = new Method();
+        $method->user_id = $user1->id;
+        $method->type = Method::TYPE_EMAIL;
+        $method->value = 'email-1456765289@domain.org';
+        if ( ! $method->save()) {
+            $this->fail('Failed to save Method when testing default values: ' .
+                print_r($method->getFirstErrors(), true));
+        }
+
+        $this->assertNotNull($method->id);
+        $this->assertEquals(32, strlen($method->uid));
+        $this->assertEquals(0, $method->verified);
+        $this->assertNull($method->verification_code);
+        $this->assertEquals(0, $method->verification_attempts);
+        $this->assertNull($method->verification_expires);
+        $this->assertNotNull($method->created);
+    }
+
+    public function testGetMaskedValuePhone()
+    {
+        $method = $this->methods('method1');
+        $this->assertEquals('1,1234567890', $method->getMaskedValue());
```

Comment:
Is it just me, or does this test prove that `getMaskedValue()` *doesn't* work? It seems to be asserting that the returned value is not masked. If this is just because we haven't implemented that masking logic yet, it might be better to assert that they're not equal, then mark this test as skipped for now.